src/inverseRobot.js

```diff
@@ -8,6 +8,16 @@
 
 function inverseRobot(robot) {
   // write code here
-}
+  const inverse = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(inverse, robot[key])) {
```

Comment:
Consider using 'hasOwnProperty' on the 'inverse' object directly instead of using 'Object.prototype.hasOwnProperty.call'.